packages/docs/pages/.vitepress/theme/custom.css

```diff
@@ -118,3 +118,8 @@
 :root {
   overflow-y: scroll;
 }
+
+.VPTeamMembersItem .avatar-img {
+  top: 50%;
+  transform: translateY(-50%);
```

Comment:
@farukaydin I had to style the avatar as the salesforce logo was not vertically centered.